assets/cluster-bootstrap/cluster-network-01-crd.yaml

```diff
@@ -3,7 +3,7 @@
 # This is the advanced network configuration CRD
 # Only necessary if you need to tweak certain settings.
 # See https://github.com/openshift/cluster-network-operator#configuring
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
The entire CRD needs to be updated to support the GA API.  We this CRD is applied, it will fail:
```
vagrant@verify-cluster:~$ kubectl apply -f crd.new.yaml 
The CustomResourceDefinition "networks.operator.openshift.io" is invalid: spec.versions[0].schema.openAPIV3Schema: Required value: schemas are required
```

Reply:
Updated the entire CRD to support schema's.